java/arcs/core/entity/Entity.kt

```diff
@@ -42,7 +42,7 @@ interface Entity : Storable {
         ttl: Ttl = Ttl.Infinite()
     )
 
-    fun serialize(): RawEntity
+    fun serialize(restrictedSchema: Schema? = null): RawEntity
```

Comment:
Proposal: Maybe instead of "restrictedSchema" we could name this "storeSchema". I think it would be more readable to someone who is not familiar with policy and ingress restricting. They would read the code and think "Oh, to we only serialize the fields that are on the store type, makes sense". What do you think?

Reply:
renamed